my-use-case/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "my-use-case"
+version = "0.0.1"
+description = ""
+authors = ["Felix Schaumann <felix.schaumann@datatonic.com>"]
+license = "MIT"
+readme = "README.md"
+homepage = ""
+repository = ""
+keywords = []
+classifiers = []
+packages = [
+    { include = "src" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.7.1"
+kfp = "1.8.9"
+shared_pipeline_components = { path = "../shared-pipeline-components/", develop = false }
```

Comment:
develop = true maybe?

Reply:
ideally only power users need dev requirements for contributing to shared-pipeline-components, end-users can focus on adding business logic to my-use-case.